src/components/app/DocumentList.js

```diff
@@ -186,7 +186,9 @@ class DocumentList extends Component {
             this.setState({
                 cachedSelection: null
             }, () => {
-                dispatch(setListIncludedView());
+                if (includedView.windowType === 'pickingSlot') {
```

Comment:
@teosarca 
Yes, this piece of code looks very weird.

Please let me explain.

1. Before my changes this line was intended to reload includedView contents when user clicks on another product line (on left side).
2. But when we added overriding includedView upon click on "Select HU" action we've get triggering of this method call when we actually just replacing the includedView with a new one.
3. So your assumption - "an included view shall be opened" then it shall be opened no matter what" is valid and it will work in the same way. The only exception - pickingSlot window, which has a bit different data flow and needs to clear contents (see line 189 before my changes) and then load new list of slots.

In fact this is workaround to avoid more massive changes in DocumentList.